### PR TITLE
Blue-green deploys.

### PR DESCRIPTION
**Pull request by jmcarp**
* Make blue and green copies of apps in manifests.
* Detect active version of app on push.
* Add route un/mapping after push.

### CODE REVIEW

**Comment by jmcarp** [comment]
Needs some discussion before deploy (see https://github.com/18F/openFEC-web-app/pull/191#issuecomment-103269504 for details).

**Comment by arowla** [comment]
See #796

**Comment by arowla** [comment]
I'm assuming we want to wait on #796 so we ensure our blue-green deploys go off without an env-var hitch.

**Comment by jmcarp** [comment]
Agree.

File: manifest_prod.yml [hunk 1]
```diff
@@ -6,10 +6,23 @@ env:
...
   path: .
   buildpack: "https://github.com/ddollar/heroku-buildpack-multi.git"
   host: fec-prod-api
+  services:
+    - fec-creds-prod
+  env:
+    PRODUCTION: True
+    NEW_RELIC_APP_NAME: OpenFEC API (production)
+    NEW_RELIC_LOG: stdout
+    WEB_CONCURRENCY: 4
```
Comment (arowla) [inline manifest_prod.yml]: Can we also list somewhere here that we need the creds service to have (manually) set:

```
# NEW_RELIC_LICENSE_KEY
# SQLA_CONN
```

**Pull request merged**